src/screens/WhotoFollow/elements/UserInfo.js

```diff
@@ -39,34 +41,34 @@ const styles = StyleSheet.create({
     flex: 1
   },
   tinyLogo: {
-    width: 45,
-    height: 45,
-    borderRadius: 48,
-    marginLeft: 3,
-    marginTop: 3
+    width: dimen.normalizeDimen(45),
+    height: dimen.normalizeDimen(45),
+    borderRadius: dimen.normalizeDimen(48),
+    marginLeft: dimen.normalizeDimen(3),
+    marginTop: dimen.normalizeDimen(3)
   },
   containerTextCard: {
     flexDirection: 'column',
     alignItems: 'center',
-    marginLeft: 8,
+    marginLeft: dimen.normalizeDimen(8),
     flex: 1
   },
   textFullName: {
     fontFamily: 'Poppins',
     fontStyle: 'normal',
     fontWeight: 'bold',
-    fontSize: 14,
-    color: '#000000',
-    lineHeight: 21,
+    fontSize: normalizeFontSize(14),
+    color: COLORS.black000,
+    lineHeight: normalizeFontSize(21),
     alignSelf: 'flex-start'
   },
   textUsername: {
-    fontSize: 14,
-    color: '#000000',
-    lineHeight: 21,
+    fontSize: normalizeFontSize(14),
+    color: COLORS.gray500,
+    lineHeight: normalizeFontSize(21),
     alignSelf: 'flex-start',
     width: '100%',
-    marginRight: 16
+    marginRight: dimen.normalizeDimen(16)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are also beneficial for maintainability and consistency. By using the `normalizeDimen` and `normalizeFontSize` functions, you ensure that the dimensions and font sizes used in the application are consistent across different screen sizes and resolutions. Also, using `COLORS` from the `utils/theme` file helps to maintain a consistent color scheme throughout the app.
```diff
-    width: 45,
-    height: 45,
-    borderRadius: 48,
-    marginLeft: 3,
-    marginTop: 3
+    width: dimen.normalizeDimen(45),
+    height: dimen.normalizeDimen(45),
+    borderRadius: dimen.normalizeDimen(48),
+    marginLeft: dimen.normalizeDimen(3),
+    marginTop: dimen.normalizeDimen(3)

-   fontSize: 14,
-   color: '#000000',
+   fontSize: normalizeFontSize(14),
+   color: COLORS.black000,

-   fontSize: 14,
-   color: '#000000',
+   fontSize: normalizeFontSize(14),
+   color: COLORS.gray500,
+   marginRight: dimen.normalizeDimen(16)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->